scripts/.eslintrc.json

```diff
@@ -1,7 +1,13 @@
 {
+  "env": {
+    "node": true
```

Comment:
This is the local .eslintrc.json for the scripts folder, container build scripts that run only in node.